htcosmos/make_parquet.py

```diff
@@ -65,6 +65,46 @@ def tlog(msg):
     now = datetime.now().strftime('%X.%f')
     print(f'{now} {msg}')
 
+def tlog_flush(msg):
+    now = datetime.now().strftime('%X.%f')
+    print(f'{now} {msg}', flush=True)
+
+# create progress file and tlog the same message, also flush output
+def set_progress(filename, msg):
+    now = datetime.now().strftime('%X.%f')
+    line = f'{now} {msg}'
+    with open(filename, 'w') as wf:
+        print(line, file=wf, flush=True)
+    print(line, flush=True)
+
+def clear_progress(filename):
```

Comment:
Same here. I think I like having docstrings for anything that can create or remove files